ProjectSidewalk/SidewalkWebpage
# Added a level of 2-means clustering and other small updates to clustering code

This adds a step of 2-means clustering, to label clustering for ground truth labels. As a heuristic, when we get to a cluster that has more than 3 labels of a certain label type, we assume that this means that there are two "issues" present there (at least), so we are now doing a 2-means clustering there to split that cluster into 2 clusters. As a result, I have increased the clustering threshold distance from 5 meter diameter to 7.5 meter diameter as default.

I also made a few updates so that clustering will work with the changes that @maddalihanumateja has made to how HITs are assigned/named.

I also made a couple minor changes to the clustering code's style/formatting/etc.

To test:
* audit a route three times with different turker ids (but keep the hit id the same)
* go to the endpoint /clustering
* enter the route id and hit id of the audits you just did and click the Run Clustering button
* check that there are new entries in the three clustering_session tables
* go to the endpoint /gtresolution
* enter the clustering_sesssion_id of that clustering that you ran, and go through the gt resolution tool to make sure that things look okay
